main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Good job making sure the inputs repeat to allow a user to input multiple stories! This initial value and while condition will only allow 8 loops to occur. Changing the initial value of `loop` to 0 and the while condition to `loop < 10` will allow this code to repeat the correct number of times.